tests/test-top-bdii

```diff
@@ -21,63 +21,59 @@ sleep 400
 echo -n "Checking that the BDII is running ... "
 ${command} >/dev/null 2>/dev/null
 if [ $? -eq 32 ]; then
-    echo "FAIL"
-    RETVAL=1
+	echo "FAIL"
+	RETVAL=1
 else
-    echo "OK"
-fi 
+	echo "OK"
+fi
 
 echo -n "Checking resource BDII publication ... "
 bind="mds-vo-name=resource,o=grid"
 filter="(&(objectClass=GlueService)(GlueServiceType=bdii_top))"
-${command} ${bind} '${filter}' >/dev/null 2>/dev/null
-if [ $? -gt 0 ]; then
-    echo "FAIL"
-    RETVAL=1
+if ! ${command} ${bind} "${filter}" >/dev/null 2>/dev/null; then
```

Comment:
shouldn't this be if [ ! ${command} ...]?

Reply:
no :)

Reply:
See the POSIX definition of `if`: 
https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_09_04_07.

The `[]` are basically a shortcut for using the `test` command, see https://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html#tag_20_128

Reply:
And the usage of `!` is here: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_09_02

Reply:
ok thanks! i understand now how it works:-)